### ISSUE TITLE
primaryKey: true fails for camelCased attribute

### ISSUE THREAD

**Issue by ofekd** [balderdashy/sails-postgresql#121]
Using this model:

```javascript
module.exports = {
    attributes: {
        someName: {
            type: 'integer',
            required: true,
            primaryKey: true
        }
    },
    autoPK: false
};
```
I get this error on first lift:

```
error: error: A hook (`orm`) failed to load!
error: error: Error (E_UNKNOWN) :: Encountered an unexpected error
error: column "somename" named in key does not exist
    at Connection.parseE (/home/webapp/baronet/node_modules/sails-postgresql/node_modules/pg.js/lib/connection.js:534:11)
    at Connection.parseMessage (/home/webapp/baronet/node_modules/sails-postgresql/node_modules/pg.js/lib/connection.js:361:17)
    at Socket.<anonymous> (/home/webapp/baronet/node_modules/sails-postgresql/node_modules/pg.js/lib/connection.js:105:22)
    at Socket.emit (events.js:95:17)
    at Socket.<anonymous> (_stream_readable.js:764:14)
    at Socket.emit (events.js:92:17)
    at emitReadable_ (_stream_readable.js:426:10)
    at emitReadable (_stream_readable.js:422:5)
    at readableAddChunk (_stream_readable.js:165:9)
    at Socket.Readable.push (_stream_readable.js:127:10)
    at TCP.onread (net.js:528:21)

Details:  error: column "somename" named in key does not exist

error: error: Error (E_UNKNOWN) :: Encountered an unexpected error
error: column "somename" named in key does not exist
    at Connection.parseE (/home/webapp/baronet/node_modules/sails-postgresql/node_modules/pg.js/lib/connection.js:534:11)
    at Connection.parseMessage (/home/webapp/baronet/node_modules/sails-postgresql/node_modules/pg.js/lib/connection.js:361:17)
    at Socket.<anonymous> (/home/webapp/baronet/node_modules/sails-postgresql/node_modules/pg.js/lib/connection.js:105:22)
    at Socket.emit (events.js:95:17)
    at Socket.<anonymous> (_stream_readable.js:764:14)
    at Socket.emit (events.js:92:17)
    at emitReadable_ (_stream_readable.js:426:10)
    at emitReadable (_stream_readable.js:422:5)
    at readableAddChunk (_stream_readable.js:165:9)
    at Socket.Readable.push (_stream_readable.js:127:10)
    at TCP.onread (net.js:528:21)

Details:  error: column "somename" named in key does not exist
```

Changing someName to somename works.

I also tested with sails-disk and both cases were fine

I'm on sails 0.10.5, sails-postgresql 0.10.9

**Comment by eduardoboss** [comment]
Same here with Sails 0.11.0

**Comment by sailsbot** [comment]
Thanks for posting, @ofekd.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!

**Comment by devinivy** [comment]
Active bug– reopening.

**Comment by ofekd** [comment]
I have upgraded to sails-postgres 0.10.17 and it didn't raise an error. So I did some tests and it looks like it was fixed in the 0.10.15 release. No idea which commit is responsible, though. I can guess but it would be unwise :-)

It'll be nice if someone else will check it so we can be sure it's fixed. Preferably someone who bumped into the issue before.

**Comment by particlebanana** [comment]
Seems fixed. If this is confirmed to still exists a new issue can be opened with a failing test. This works in all my test.
